exercises/perfect-numbers/runtests.jl

```diff
@@ -0,0 +1,42 @@
+using Test
+include("example.jl")
```

Comment:
```suggestion
include("perfect-numbers.jl")
```

The student doesn't have access to `example.jl` when solving the exercise, so this would fail. To run the tests locally, use `julia runtests.jl perfect-numbers` at the root of this repo. It will automatically load the example instead of the stub file.